android/src/main/java/com/clevertap/clevertap_plugin/CleverTapPlugin.java

```diff
@@ -42,6 +42,10 @@ public class CleverTapPlugin implements ActivityAware, FlutterPlugin {
 
     private static final Handler mainHandler = new Handler(Looper.getMainLooper());
 
+    private final Runnable runnable = () -> {
```

Comment:
Can we have a more suitable name for this field?

Reply:
renamed this field.